receiver/aerospikereceiver/metadata.yaml

```diff
@@ -0,0 +1,125 @@
+    extended_documentation: Aggregate of Aerospike Metrics client_connections_closed, client_connections_opened, fabric_connections_closed, fabric_connections_opened, heartbeat_connections_closed, heartbeat_connections_opened
+    unit: '{connections}'
+    attributes: [connection_type, connection_op]
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  aerospike.node.connection.open:
+    enabled: true
+    description: Number of open connections to the node
+    extended_documentation: Aggregate of Aerospike Metrics client_connections, fabric_connections, heartbeat_connections
+    unit: '{connections}'
+    attributes: [connection_type]
+    gauge:
+      value_type: int
+  aerospike.namespace.memory.usage:
+    enabled: true
+    description: Memory currently used by each component of the namespace
+    extended_documentation: Aggregate of Aerospike Metrics memory_used_data_bytes, memory_used_index_bytes, memory_used_set_index_bytes, memory_used_sindex_bytes
+    unit: By
+    attributes: [namespace_component]
+    sum:
+      value_type: int
+      monotonic: false
+      aggregation: cumulative
+  aerospike.namespace.memory.free:
+    enabled: true
+    description: Percentage of the namespace's memory which is still free
+    extended_documentation: Aerospike metric memory_free_pct
+    unit: '%'
+    gauge:
+      value_type: int
+  aerospike.namespace.disk.available:
+    enabled: true
+    description: Minimum percentage of contiguous disk space free to the namespace across all devices
+    unit: '%'
+    gauge:
+      value_type: int
+  aerospike.namespace.span.count:
+    enabled: true
+    description: Number of scan operations performed on the namespace
+    extended_documentation: Aggregate of Aerospike Metrics scan_aggr_abort, scan_aggr_complete, scan_aggr_error, scan_basic_abort, scan_basic_complete, scan_basic_error, scan_ops_bg_abort, scan_ops_bg_complete, scan_ops_bg_error, scan_udf_bg_abort, scan_udf_bg_complete, scan_udf_bg_error
+    attributes: [scan_type, scan_result]
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  aeropsike.namespace.transaction.count:
+    enabled: true
+    description: Number of transactions performed on the namespace
+    extended_documentation: Aggregate of Aerospike Metrics client_delete_error, client_delete_filtered_out, client_delete_not_found, client_delete_success, client_delete_timeout, client_read_error, client_read_filtered_out, client_read_not_found, client_read_success, client_read_timeout, client_udf_error, client_udf_filtered_out, client_udf_not_found, client_udf_success, client_udf_timeout, client_write_error, client_write_filtered_out, client_write_not_found, client_write_success, client_write_timeout
+    attributes: [transaction_type, transaction_result]
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
```

Comment:
These two look like they're missing units;

Also should have a newline at the end